### PR TITLE
Creating tribute

### PR DESCRIPTION
**Pull request by sodabarauf**
this project is about making a tribute page, in this project I did these things:
#first of all, I followed the GitHub workflow which include creating repository, creating development branch and feature branch, clone repository with local machine, work on feature branch, set up linters config, work and commit, push commits to GitHub....
#i added two new files index.html and style.css.
#i linked HTML with CSS file
#i added main tag, h1 tag, h3 tag, and h5 tag to the index.html file.
#i added div tags, img tag, fig caption tag, ul and li tags.
#i added some styles in the CSS file.
#i added the linters config of html and css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        #image {
+          max-width: 100%;
+          height: auto;
+        }
+
+        img {
+            display: block;
+        }
+        
+    </style>
```
Comment (ahzamir) [inline index.html]: - You have a separate file for adding your style codes so remove and separate the style codes from your HTML file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    <li><strong>Death and Legacy: </strong>Rūmī lived for a short while after completing the Mas̄navī.and died on 17 December 1273. He always remained a respected member of Konya society,His burial procession, according to one of Rūmī’s contemporaries, was attended by a vast crowd of people of many faiths and nationalities. His mausoleum, the Green Dome, is today a museum in Konya; it is still a place of pilgrimage, primarily for Turkish Muslims.</li>
+                    </ul>
+                    </div>
+                
+                <h5>
+                        If you have time, you should read more about this incredible person being on his<a id="tribute-link" target="_blank" href="https://www.WIKIpedia.com" rel="noopener">WIKIpedia</a>
+                </h5>
+          </main>
+    
+</body>
```
Comment (ahzamir) [inline index.html]: - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes all checks on freecodecamp website on the tribute page project, but I added some inline comments **(Check below)** for you to implement for clean code.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
image
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean.

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by sodabarauf** [comment]
@ahzamir, sir thanks from your feedback, beside external CSS I added internal CSS because in freecodecamp.org, there showed error, so I added, now removed the internal CSS and hope be acceptable.
also, I cleaned the code according to the requirement which you have provided to me.
thank you very much sir.

**Comment by ahzamir** [comment]
@sodabarauf 
- Still, your code is not clean in the HTML file also delete the `HEAD>>>>>>>` and other unnecessary lines from your code
- And for cleaning you code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!

**Comment by sodabarauf** [comment]
@ahzamir 
sir, thank you. 
now, please review this.

**Comment by sodabarauf** [comment]
@ahzamir sir please review my changes which i worked again and provide feedback if possible,
Because by adding these changes I think it has caused some problem.

**Review by ahzamir — APPROVED** [review]
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
Screenshot 2024-05-22 114625



# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by sodabarauf** [comment]
@ahzamir sir, I appreciate from your struggles in this part with us.
thank you.

**Pull request merged**